server/src/internalClusterTest/java/org/opensearch/indices/recovery/IndexPrimaryRelocationIT.java

```diff
@@ -101,7 +98,7 @@ public void run() {
             ClusterHealthResponse clusterHealthResponse = client().admin()
                 .cluster()
                 .prepareHealth()
-                .setTimeout(TimeValue.timeValueSeconds(60))
+                .setTimeout(TimeValue.timeValueSeconds(120))
```

Comment:
Why this change?

Reply:
the test was flaky, got soem timeout errors so increase it to validated and it worked, i can revert if but might be a fix for flakyness

Reply:
It is this one https://github.com/opensearch-project/OpenSearch/issues/9191 ? We could address that separately

Reply:
yes, okay i can revert it and try to address separately